### ISSUE TITLE
The francetv downloader is no longer operational

### ISSUE THREAD

**Issue by pavel167** [yt-dlp/yt-dlp#9350]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

France

### Provide a description that is worded well enough to be understood

Hello,
Perhaps francetv modified access to videos, as since some days they aren't available for downloading. In the same time they are Ok onliny playing.
I investigated the code in ../extractor/francetv.py: instruction `dinfo = self._download_json` fails to get **dinfo**.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
(env_video_dnld) pavel@MISSURI:~/env_video_dnld$ yt-dlp -F https://www.france.tv/documentaires/art-culture/3004513-bacri-comme-un-air-de-famille.html
[FranceTVSite] Extracting URL: https://www.france.tv/documentaires/art-culture/3004513-bacri-comme-un-air-de-famille.html
[FranceTVSite] 3004513-bacri-comme-un-air-de-famille: Downloading webpage
[FranceTV] Extracting URL: francetv:508fc41b-3ae2-4523-965a-da3140cf302d
[FranceTV] 508fc41b-3ae2-4523-965a-da3140cf302d: Downloading desktop video JSON
WARNING: [FranceTV] Unable to download JSON metadata: HTTP Error 400: Bad Request
[FranceTV] 508fc41b-3ae2-4523-965a-da3140cf302d: Downloading mobile video JSON
WARNING: [FranceTV] Unable to download JSON metadata: HTTP Error 400: Bad Request
ERROR: 'NoneType' object has no attribute 'strip'
```

**Comment by bashonly** [comment]
Duplicate of #9323

Closed by 9749ac7fecbfda391afbadf2870797ce0e382622

update to nightly
```
yt-dlp --update-to nightly
```
or

```
python3 -m pip install -U --pre yt-dlp
```

**Comment by pavel167** [comment]
Ok, thanks.
